ui/README.md

```diff
@@ -26,6 +28,19 @@ Before committing, be sure to run `make` to generate a non-debug version of
 `embedded.go`. This is enforced by our build system, but forgetting to do this
 will result in wasted time waiting for the build.
 
+## Watch/Livereload
+If you want to use gulp watch, you'll need to install gulp globally
+([see dependencies](####dependencies).) Then use `gulp` to start auto-recompiling
```

Comment:
Aren't the #'s necessary in order to make a proper internal link?  It's a github extension to markdown.